### DISCUSSION TITLE
Video overview of setup

### DISCUSSION THREAD

**Discussion by mcobr** [mtalcott/google-photos-deduper#26] [Q&A] [answered]
Hi there - I only dabble in software and am trying to set up your tool.

It would be super useful if there was a video walking through the python and Docker setup stuff!

**Comment by real-lmcampos** [comment]
Same here

**Comment by brianhughesspe** [comment, marked as answer]
Not a video, but just posted a pretty detailed explainer here: [https://github.com/mtalcott/google-photos-deduper/discussions/46](https://github.com/mtalcott/google-photos-deduper/discussions/46)
